src/libstore/build.cc

```diff
@@ -3644,10 +3644,10 @@ void DerivationGoal::registerOutputs()
            hash). */
         std::string ca;
 
-        if (fixedOutput) {
+        if (i.second.hashAlgo != "") {
```

Comment:
Can't this be replaced by `DtAxisCA & derivationType` ?

Reply:
Ah, this too was a remnant getting ahead to encoding CA derivations as having outputs with a hash algo but not a hash.

Reply:
We'll remove for now.